pkg/engine/task/task_kudo_operator.go

```diff
@@ -137,7 +137,7 @@ func instanceResource(instanceName, operatorName, operatorVersionName, namespace
 		},
 		Status: v1beta1.InstanceStatus{},
 	}
-	if err := controllerutil.SetControllerReference(owner, instance, scheme); err != nil {
+	if err := controllerutil.SetOwnerReference(owner, instance, scheme); err != nil {
```

Comment:
This is the actual change: with `SetOwnerReference ` reconciliation will be called for the parent instance